packages/flutter_tools/flutter_analysis_options

```diff
@@ -25,28 +27,29 @@ analyzer:
     strong_mode_down_cast_composite: ignore
     # we allow having TODOs in the code
     todo: ignore
+  exclude:
+    - 'bin/**'
 
 linter:
   rules:
-    # these are in the same order as http://dart-lang.github.io/linter/lints/
-    # to make maintenance easier
+    # these rules are documented on the Dart Lint rules page
+    # http://dart-lang.github.io/linter/lints/
```

Comment:
Are they no longer in the same order? Having them in the same order is hugely helpful.

If they're in the same order, we should leave a comment saying so so that people don't change the order.